enricher/epss/epss.go

```diff
@@ -0,0 +1,391 @@
+package epss
```

Comment:
Can you add a package-level Go doc to quickly explain the purpose of this? For example, here's the CVSS enricher:

```
// Package cvss provides a cvss enricher.
package cvss
```